src/main/java/io/github/wimdeblauwe/errorhandlingspringbootstarter/AbstractErrorHandlingConfiguration.java

```diff
@@ -1,19 +1,19 @@
 package io.github.wimdeblauwe.errorhandlingspringbootstarter;
 
-import io.github.wimdeblauwe.errorhandlingspringbootstarter.handler.BindApiExceptionHandler;
-import io.github.wimdeblauwe.errorhandlingspringbootstarter.handler.HttpMessageNotReadableApiExceptionHandler;
-import io.github.wimdeblauwe.errorhandlingspringbootstarter.handler.TypeMismatchApiExceptionHandler;
+import io.github.wimdeblauwe.errorhandlingspringbootstarter.handler.*;
 import io.github.wimdeblauwe.errorhandlingspringbootstarter.mapper.ErrorCodeMapper;
 import io.github.wimdeblauwe.errorhandlingspringbootstarter.mapper.ErrorMessageMapper;
+import io.github.wimdeblauwe.errorhandlingspringbootstarter.mapper.GraphQlErrorMapper;
 import io.github.wimdeblauwe.errorhandlingspringbootstarter.mapper.HttpStatusMapper;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
 import org.springframework.context.annotation.Bean;
 
 public abstract class AbstractErrorHandlingConfiguration {
     @Bean
     @ConditionalOnMissingBean
-    public LoggingService loggingService(ErrorHandlingProperties properties) {
-        return new LoggingService(properties);
+    public LoggingService loggingService(ErrorHandlingProperties properties,
```

Comment:
I would rather not have any graphql related classes in this class. It would be better to have a dedicated subclass.

Reply:
I see. Do you want this Configuration class to be a separate type of `ServletErrorHandlingConfiguration` specific to GraphQL configuration? Would there be a requirement to mark it as optional? Such way in properties with `@ConditionalOnProperty(value = "error.handling.graphql.enabled", matchIfMissing = false)`?